evaluator_checkpoint_folder: /workspace/sources/6dof-graspnet/checkpoints/ACRONYM_Evaluator/
gradient_based_refinement: False
npy_folder: /demo/data/
threshold: -1
vae_checkpoint_folder: /workspace/sources/6dof-graspnet/checkpoints/ACRONYM_GAN/



# visualization
vis:
  grasp_sampling : 0
  tf_images: 0
  grasp_candidates: 0
  elite_grasps: 0
  grasp_ranking: 0
  grasp_plan: 0
  final_grasp: 0

  vmin: 0.5
  vmax: 0.8

vis:
  rgbd_state: 0
  cropped_rgbd_image: 0
  color_image: 0
  depth_image: 0
  segmask: 0
  best_grasp: 1

# image pre-processing before input to policy
inpaint_rescale_factor: 0.5


